### PR TITLE
Branch dg

### PR DESCRIPTION
**Pull request by amoonguss1**
This PR will update the use cases of the DG and remove grammatical errors. (might miss a few)
This closes issue #244

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -451,132 +457,147 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
-**Extensions**
+**Extensions:**
 
-* 2a. The list is empty.
-  Use case ends
-* 2b. The given second command is invalid e.g "ascending".
-    * 2b1. TechTrack shows an error message.
-      Use case resumes at step 2.
+* 3a. Job does not exist.
```
Comment (dsja612) [inline docs/DeveloperGuide.md:520]: Should these be 3a and 3b or 2a and 2b? 🤔

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -451,132 +457,147 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
+**Use case: View a job**
 **MSS**
 
-1. User opens the application
-2. User enters the “salary” command followed by either "asc" or "desc"
-3. UI display the list of jobs sorted by salaries in either ascending or descending orderParser with indexes.
+1. User opens the application.
+2. TechTrack shows the list of jobs.
+3. User enters the “view” command.
```
Comment (dsja612) [inline docs/DeveloperGuide.md:513]: This should be "view {index}"

**Review by dsja612 — CHANGES_REQUESTED** [review]
Gj fixing most stuff, lgtm after a few nits

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (amoonguss1) [reply docs/DeveloperGuide.md]: Updated in the new commit.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (amoonguss1) [reply docs/DeveloperGuide.md]: Updated in the new commit

**Review by dsja612 — APPROVED** [review]
LGTM!

**Pull request merged**